packages/create-neutrino-project/yarn.lock

```diff
@@ -0,0 +1,1621 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This `yarn.lock` shouldn't exist. Are you using an old version of yarn that doesn't support workspaces? Please remove from the PR :-)

Reply:
Good catch, yeah this is from moving the project from a standalone repo to be included in the monorepo.